Month2/Pointers_Exercise/Section1Pointers/Exercise4/Exercise4.cpp

```diff
@@ -0,0 +1,11 @@
+#include <iostream>
+int main()
+{
+	int VariableA = 10;
+	std::cout << "The value of my variable is: " <<VariableA << std::endl;
+	int* Ptr = &VariableA;
+	(* Ptr)++;
+	int* Ptr2 = &VariableA;
+	(* Ptr2++);
+	std::cout << "The value of my variable after the pointer stuff I did is: " << VariableA << std::endl;
```

Comment:
Did you test this? Doesn't seem like it would give a correct result. You're not doing anything to the value of `VariableA` with the operation done to `Ptr2`